tests/python/test_do_not_support_generated_columns.py

```diff
@@ -0,0 +1,78 @@
+import tempfile
+
+import pytest
+
+from python_hooks.do_not_support_generated_columns import check_file
+from python_hooks.do_not_support_generated_columns import filter_files
+
+
+@pytest.mark.parametrize(
+    "file_content, expected",
+    [
+        (  # true negative
+            """
+CREATE TABLE IF NOT EXISTS some.table
+(
+id  INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+);
```

Comment:
could use some indentation here

Reply:
i unindented because it makes test output more readable, this has to do with the way mark.parametrize works. 
formatter was ok with it so i i went with ahead.